build.sbt

```diff
@@ -113,7 +108,7 @@ lazy val extras = project
   .settings(sonatypeSettings: _*)
   .settings(
     name := "sbt-jmh-extras",
+    scalaVersion := "2.12.2",
```

Comment:
We do benchmark on Scala 2.13.x, which is a reason to avoid a Scala dependency for this module.